scripts/ros_install.bash

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e  # exit on first error
+ROS_VERSION="indigo"
+ROS_BASH="/opt/ros/$ROS_VERSION/setup.bash"
+ROS_PACKAGES_URL='http://packages.ros.org/ros/ubuntu'
+APT_KEYS_URL='hkp://pool.sks-keyservers.net:80'
+APT_TARGETS="$(lsb_release -sc) main"
+SOURCES_LIST_TARGET='/etc/apt/sources.list.d/ros-latest.list'
+
+
+install()
+{
+	# update sources.list and add apt-keys
+	echo "deb $ROS_PACKAGES_URL $APT_TARGETS" > $SOURCES_LIST_TARGET
+	apt-key adv --keyserver $APT_KEYS_URL --recv-key "0xB01FA116"
+
+	# update apt and install ros
+	apt-get update
+	apt-get install -y ros-$ROS_VERSION-desktop-full
+
+	# initialize rosdep
+	rosdep init
+	rosdep update
+
+	# env setup
+	echo "source /opt/ros/$ROS_VERSION/setup.bash" >> $HOME/.bashrc
+
+	# install ros
```

Comment:
It is not necessarily at current I don't think, it was included in the ROS install instructions. But this script could be used to speed up new-comers into using ROS from a fresh install of Ubuntu, that's why I included it. It can be removed for this repo.